apps/patient/src/views/Pharmacy.tsx

```diff
@@ -422,10 +422,13 @@ export const Pharmacy = () => {
     }
   }, [location]);
 
-  const patientAddressInAustinTX =
+  const deliveryAddressInAustinTX =
     order?.address?.city === 'Austin' && order?.address?.state === 'TX';
```

Comment:
Can we check the list of zipcodes we have?

Reply:
ah, yep added